zjit/src/cruby_methods.rs

```diff
@@ -83,7 +174,12 @@ pub fn init() -> Annotations {
     annotate!(rb_cNilClass, "nil?", types::TrueClass, no_gc, leaf, elidable);
     annotate!(rb_mKernel, "nil?", types::FalseClass, no_gc, leaf, elidable);
 
+    annotate_builtin!(rb_mKernel, "Float", types::Flonum);
```

Comment:
Does `types::Flonum` mean the same as just `Float`?
I would think `Kernel#Float` can return either a flonum (an immediate/tagged pointer) or a heap-allocated Float, depending on the argument, and so it seems wrong to say the return type is "flonum" when it can be either.
(same issue in the PR description)

Reply:
Thanks. I'll open a PR for it